src/components/Header/Header.styles.js

```diff
@@ -73,15 +73,6 @@ export const StyledNavLink = styled('a')`
   display: flex;
 `
 
-export const WhatsNewLink = styled(StyledNavLink)`
```

Comment:
OOO, thanks for fully removing this too. These orphan styles often enough get ignored, and linters and such dont catch it.